.vscode/scripts/runtime/k3d/build_vehicleapp.sh

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2022 Robert Bosch GmbH and Microsoft Corporation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../../../.." )
+APP_NAME=$(cat $ROOT_DIRECTORY/app/AppManifest.json | jq .[].Name | tr -d '"')
+DOCKERFILE_FILE="$(cat $ROOT_DIRECTORY/app/AppManifest.json | jq .[].Dockerfile | tr -d '"')"
+
+if [ -n "$HTTP_PROXY" ]; then
+    echo "Building image with proxy configuration"
+
+    cd $ROOT_DIRECTORY
+    DOCKER_BUILDKIT=1 docker build \
```

Comment:
Do you have same issue behind proxy as python has? No logs available on k3d?

Reply:
Since the script is identical to the python one I would assume the issues remain the same.